dunstctl

```diff
@@ -31,6 +31,7 @@ show_help() {
 	                                    history with given ID.
 	  is-paused                         Check if dunst is running or paused
 	  set-paused [true|false|toggle]    Set the pause status
+	  rules                             Displays configured rules (in JSON)
```

Comment:
I know the history command already formats in json by default, but I'm wondering if it's not better to format in human-readable format by default and allow for a json toggle.

Reply:
This should be easily feasible.

My current idea is to have:
```
	  rules (as-json|human-readable)    Displays configured rules
```
with `human-readable` being the default when omitted. I don't know for sure, what's the correct notation for denoting an optional argument (or should it be mandatory, though I think it would be nice to have a default for interactive usage) and just used parentheses as an example.

How should the human-readable form look like? One could just emit ini-style information, basically recreating the original input.

Reply:
I would prefer to use an argument like `--json` for json output. And it should definitely not be required, for the reason you mentioned. 

Optional commands are put in [square brackets].

Reply:
I don't know what the best human-readable output is, but I would just look at what's easy to create. On second thought, the pretty json output might be good enough as well

Reply:
I currently don't have a good test setup to see, so you'll have to judge for yourself what's best

Reply:
Sure, I can do `--json` as well. I just wanted to be consistent how other commands are used.

> Optional commands are put in [square brackets].
Yes, that's how I would've expected it as well, but the usage output doesn't use it this way consistently. `history-pop` and `history-rm` look the same, but just `history-pop` uses it as an optional argument. `set-paused` and `rule` also use them for mandatory arguments.

Reply:
> Sure, I can do `--json` as well. I just wanted to be consistent how other commands are used.

You don't have to do it just because I mentioned it. Only if you think it's a good idea. Consistency between different commands is also nice.

> > Optional commands are put in [square brackets].
> 
> Yes, that's how I would've expected it as well, but the usage output doesn't use it this way consistently. `history-pop` and `history-rm` look the same, but just `history-pop` uses it as an optional argument. `set-paused` and `rule` also use them for mandatory arguments.

Oh that's a little bit weird. You would only need one of these commands. You can fix that in a separate PR if you want